src/domain/navigation/index.js

```diff
@@ -132,8 +132,10 @@ export function parseUrlPath(urlPath, currentNavPath, defaultSessionId) {
                 segments.push(roomsSegmentWithRoom(rooms, roomId, currentNavPath));
             }
             segments.push(new Segment("room", roomId));
-            if (!urlPath.includes("/member/")) {
-                // Add right-panel segments from previous path if /member is not in url
+            const partIndex = parts.findIndex(part => part === "open-room");
+            const partsAfterRoom = parts.slice(partIndex + 1);
+            if (partsAfterRoom.length === 1) {
```

Comment:
Thanks for the changes! A bit of a nit, but I don't think we need to create an array here with slice, as we won't be using it. Can also do just:
```js
            const openRoomPartIndex = parts.findIndex(part => part === "open-room");
            // -2 for the room id after open-room and the open-room part itself
            const hasOnlyRoomIdAfterPart = openRoomPartIndex >= parts.length - 2;
            if (hasOnlyRoomIdAfterPart) {
```

Reply:
Done :heavy_check_mark: